app/views/publish_mailer/publish_email.text.erb

```diff
@@ -0,0 +1,36 @@
+<%= @edition.document_type.label %> ‘<%= @edition.title %>’
+
+<% if @edition.number > 1 %>
+  <%= t("publish_mailer.publish_email.details.update",
+    time: @status.created_at.strftime("%l:%M%P"),
+    date: @status.created_at.strftime("%d %B %Y"),
+    user: @status.created_by.name,
+  ) %>
+<% else %>
+  <%= t("publish_mailer.publish_email.details.publish",
+    time: @status.created_at.strftime("%l:%M%P"),
+    date: @status.created_at.strftime("%d %B %Y"),
+    user: @status.created_by.name,
+  ) %>
+<% end %>
+
+<%= t("publish_mailer.publish_email.delay_warning") %>
+
+<%= t("publish_mailer.publish_email.page_address") %>
+<%= EditionUrl.new(@edition).public_url %>
+
+<% if @edition.number > 1 %>
+  <% if @edition.major? %>
+    <%= t("publish_mailer.publish_email.change_note") %>
+    <%= @edition.change_note %>
+  <% else %>
+    <%= t("publish_mailer.publish_email.minor_update") %>
+  <% end %>
+<% end %>
+
+<% if @edition.published_but_needs_2i? %>
+  <%= t("publish_mailer.publish_email.2i_warning") %>
+<% end %>
+
+<%= t("publish_mailer.publish_email.edit_in_app") %>
+<%= document_url(@edition.document) %>
```

Comment:
I'm not sure we gain a lot by this using translation phrases since it's a text file and would presumably be easier to edit here than through translation files. I'm imagining if a content designer was to come to edit this copy it'd be quite a complicated process where it's quite hard to determine the email copy due to the translations.

I'm thinking if we were to support multiple languages we'd probably have separate text view files for each language rather than phrases, since the logic is all around text rather than being wrapped in components/HTML.

Apologies if we already have a precedent and I missed this the first time around, what are your thoughts?

Reply:
This one is a bit unusual, as normally we're passing translations as arguments to a component, whereas here they're just inline. But this is covered in the [ADR](https://github.com/alphagov/content-publisher/blob/master/docs/adr/0004-editing-microcopy.md#decision) under 'All other strings', so I've stuck with that. 

The only place I'm aware we're not consistent with this is the [scheduling form](https://github.com/alphagov/content-publisher/blob/master/app/views/documents/show/_scheduling_form.html.erb). I think we've avoided steering content designers towards ERB files thus far - it starts to make it hard to find where a particular string is. 

Possibly this is just another occurrence of where the scope of the original card was too big, but on the other hand, it didn't bother me much putting them in.

Reply:
Yeah, I'd agree this one is unusual. To me it fits more into the realm of the guidance files where we're already sending content designers to erb files, e.g: https://github.com/alphagov/content-publisher/blame/master/app/views/new_document/guidance.govspeak.md but can also understand how it fits into the definition from the ADR. 

For the most part I'm expecting content designers have to search for a phrase whatever since they don't know the view structure.

Reply:
The ADR does also have a specific rule for "big guidance", so I'm reluctant to lump this in with that - it was more about the nature of the content, as opposed to the format it's in.

I wouldn't say this has caused me any pain, but if it's something we'd like to revisit, perhaps the best approach is to open an issue to look at it again - having rules to follow does help simplify this kind of scenario, but I remember we never really satisfied everyone.